### PR TITLE
Core/utils/body validator

### PR DESCRIPTION
**Pull request by Enrique-C**

### CODE REVIEW

File: core/api/features/steps/Common.py [hunk 1]
```diff
@@ -28,12 +29,12 @@ def step_impl(context, name_object):
 
 @step('Validates schema with "{schema}"')
 def step_impl(context, schema):
-    expected = SchemaValidator.validate(context.json_response.json(), schema)
-    assert True is expected
+    result = SchemaValidator.validate(context.json_response.json(), schema)
+    assert True is result
```
Comment (JuanCarlosMartinez4) [inline core/api/features/steps/Common.py]: improve this assert

File: core/api/features/steps/Common.py [hunk 2]
```diff
@@ -28,12 +29,12 @@ def step_impl(context, name_object):
...
-@step("Validates response body")
-def step_impl(context):
-    """
-    :type context: behave.runner.Context
-    """
+@step('Validates response body with "{expected_body}"')
+def step_impl(context, expected_body):
+    body_validator = BodyValidator()
+    result = body_validator.validate(context.json_response, expected_body, context.data_table)
+    assert True is result
```
Comment (JuanCarlosMartinez4) [inline core/api/features/steps/Common.py:39]: improve this assert

File: trello/jsonfiles/expected_board_body.json [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+    "red": "",
+    "purple": "",
+    "blue": "",
+    "sky": "",
+    "lime": "",
+    "pink": "",
+    "black": ""
+  },
+  "limits": {}
+}
```
Comment (JuanCarlosMartinez4) [inline trello/jsonfiles/expected_board_body.json]: add a new line at end of the file

File: core/api/features/steps/Common.py (hunk 2 above)
Comment (Enrique-C) [reply core/api/features/steps/Common.py:39]: It works of that way, because it doesn't be a method of a Library

**Review by JuanCarlosMartinez4 — APPROVED** [review]

**Review by Amet015 — APPROVED** [review]

**Pull request merged**